region/maridia/inner-yellow.json

```diff
@@ -588,13 +588,20 @@
                 {
                   "name": "Crouch Jump Down Grab Escape",
                   "notable": false,
-                  "requires": ["h_canCrouchJumpDownGrab"],
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "h_canCrouchJumpDownGrab",
+                    "canTrickyJump"
```

Comment:
That is true. I was debating between tricky and careful, but was thinking tricky fit a little better, as I have noticed a lot of people try and think that strat isn't possible (partly because it isn't on the left side).